### ISSUE TITLE
Model Library: Ability to update model manifest via editor

### ISSUE THREAD

**Issue by MaxJa4** [ollama/ollama#6446]
# TLDR

Direct updates of small, text-based files like parameters, template, license and system message **within the Ollama library** to avoid the time-consuming and bandwidth-heavy process of pulling, updating and pushing all tags of a model. This would simplify updates without affecting already existing possibilities with ollama-push.

# Details

Currently, when changes to a manifest/modelfile need to be made (e.g. template or parameters), all `N` tags of a model need to be pulled, updated and then pushed `N` times respectively.
This can be quite time consuming (especially for 70B+ models and with many quantizations) as well as somewhat wasteful regarding bandwidth of the Ollama library - especially since things like the template are often shared between all tags, so just one single object would need to be updated.

Perhaps updating purely text-based, small files/objects like parameters, license, system message and template could be done directly in the Ollama library, like the model readme can (by the author).  That would leave the fine-grained control via ollama-push untouched and not complicate anything there.
That's just what came to mind first though, there might be other sensible approaches as well.

Since the library is not open source as far as I'm aware, I can't help with contributing there myself, so here's a feature request to hopefully keep an eye on this topic. Many thanks!
